src/python/grapl_analyzerlib/grapl_analyzerlib/nodes/__init__.py

```diff
@@ -0,0 +1,21 @@
+from grapl_analyzerlib.nodes.process import ProcessSchema
+from grapl_analyzerlib.nodes.asset import AssetSchema
+from grapl_analyzerlib.nodes.file import FileSchema
+from grapl_analyzerlib.nodes.ip_address import IpAddressSchema
+from grapl_analyzerlib.nodes.ip_connection import IpConnectionSchema
+from grapl_analyzerlib.nodes.ip_port import IpPortSchema
+from grapl_analyzerlib.nodes.lens import LensSchema
+from grapl_analyzerlib.nodes.process_inbound_connection import ProcessInboundConnectionSchema
+from grapl_analyzerlib.nodes.process_outbound_connection import ProcessOutboundConnectionSchema
+from grapl_analyzerlib.nodes.risk import RiskSchema
+
```

Comment:
Why do we have these init_reverses here + also in each module?

Reply:
It's to ensure that regardless of how the type is imported they get init_reverse'd. It may be redundant, but yeah, that's the thought there.